src/app/footer/footer.component.html

```diff
@@ -18,12 +18,13 @@ <h3>About</h3>
     </div>
     <div class="nav-links__row">
       <div class="nav-links nav-links__how-we-can-help">
-        <h3>How we can help</h3>
+        <h3>How can we help</h3>
         <ul>
           <li><a mat-button class="nav-link" href="https://www.thebiggive.org.uk/s/charities">Charities</a></li>
           <li><a mat-button class="nav-link" href="https://www.thebiggive.org.uk/s/philanthropists">Philanthropists</a></li>
           <li><a mat-button class="nav-link" href="https://www.thebiggive.org.uk/s/individuals">Individuals</a></li>
           <li><a mat-button class="nav-link" href="https://www.thebiggive.org.uk/s/christmas-challenge">Christmas Challenge</a></li>
+          <li><a mat-button class="nav-link" href="https://www.thebiggive.org.uk/s/other-campaigns">Other Campaigns</a></li>
```

Comment:
@tbg-jasonfung casing is not consistent with SF, we should decide on a convention and stick to it.